src/_posts/gamestate.md

```diff
@@ -0,0 +1,346 @@
+---
+title: "Managing Application and Game State in A-Frame"
```

Comment:
Post is about game state itself which is a necessity rather than something optional, talking about different implementations.